common/djangoapps/nodebb/tasks.py

```diff
@@ -38,18 +38,13 @@ def handle_response(caller, task_name, status_code, response, username=None):
 
 
 @task(default_retry_delay=RETRY_DELAY, max_retries=None, routing_key=settings.HIGH_PRIORITY_QUEUE)
-def task_create_user_on_nodebb(username, user_data):
+def task_create_user_on_nodebb(username, user_data, user):
```

Comment:
please also update `task_create_user_on_nodebb` call in test `def test_user_creation(self):`  in file `common/djangoapps/nodebb/tests.py`